controllers/gatekeeper_controller.go

```diff
@@ -51,3 +119,60 @@ func (r *GatekeeperReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&operatorv1alpha1.Gatekeeper{}).
 		Complete(r)
 }
+
+func (r *GatekeeperReconciler) deployGatekeeperResources() error {
+	ctx := context.Background()
+	var err error
+	for _, a := range orderedStaticAssets {
+		assetName := staticAssetsDir + a
+		bytes, err := bindata.Asset(assetName)
+		if err != nil {
+			return err
+		}
+
+		logger := r.Log.WithValues("Gatekeeper resource", string(assetName))
+		manifest := &manifest.Manifest{}
+		err = manifest.UnmarshalJSON(bytes)
+		if err != nil {
+			return err
+		}
+
+		err = r.Create(ctx, manifest.Obj)
+		if err == nil {
```

Comment:
It's a good practice to keep the happy path on the left side (see https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88)

Reply:
It was (just as everything else) a suggestion to keep the code more idiomatic.